addons/utm/views/utm_stage_views.xml

```diff
@@ -22,6 +22,20 @@
         </field>
     </record>
 
+    <record id="utm_stage_view_form" model="ir.ui.view">
+        <field name="name">utm.stage.view.form</field>
+        <field name="model">utm.stage</field>
+        <field name="arch" type="xml">
+            <form string="Stages">
+                <sheet>
+                    <group>
+                        <field name="name" class="w-50" placeholder='e.g. "Brainstorming"'/>
```

Comment:
Hmm, would avoid using width classes on fields.

Can you solve it using regular view tags instead? Maybe by having 2 groups inside each other?